app/src/main/res/values-fr/strings.xml

```diff
@@ -107,8 +107,9 @@
     <string name="hide_filters">Filtres</string>
     <string name="hide_search">Rechercher</string>
 
-    <!--Module Fragment-->
-    <string name="no_info_provided">(aucune info transmise)</string>
+    <!--Module -->
+    <string name="no_info_provided">(Aucune info transmise)</string>
```

Comment:
No capital letter after an opening parenthesis in French. Please keep the original:
```xml
 <string name="no_info_provided">(aucune info transmise)</string>
```